e2e/transport.go

```diff
@@ -46,6 +47,37 @@ type transportHook interface {
 	Gossip(from, to pbft.NodeID, msg *pbft.MessageReq) bool
 }
 
+// encapsulates routing mapping for certain round
+type roundMetadata struct {
+	round uint64
+	// represents message routing map (sender node id to recipient node ids)
+	// e.g. A4 sends messages to [A0, A1]
```

Comment:
If you need to explain the usage of the field in a comment, it's probably not named right